final_prep/annotations.rb

```diff
@@ -16,28 +22,37 @@ def build_a_bear(name, age, fur, clothes, special_power)
     'sayings' => [greeting, power_saying, "Goodnight my friend!"],
     'is_cuddly' => true,
   }
+  # return the hash with all of the data
   return built_bear
 end
 
+# run the build_a_bear method with two different sets of inputs
 build_a_bear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares')
 build_a_bear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in')
 
 
 # FizzBuzz
 
+# create a new method that prints fizz or buzz depending on the values input. Allow three arguments, num1, num2, and range.
 def fizzbuzz(num_1, num_2, range)
+  # iterate over a range of values from 1 to range, using i as the iterator
   (1..range).each do |i|
+    # if the modulus of i by both num_1 and num_2 are exactly 0, print 'fizzbuzz' to terminal
```

Comment:
Describing a modulo in plain terms can be a good practice in technical explanation.